app/status.go

```diff
@@ -331,6 +453,12 @@ func (a *App) SaveAndBroadcastStatus(status *model.Status) {
 	a.BroadcastStatus(status)
 }
 
+func (a *App) SaveStatusSchedule(statusSchedule *model.StatusSchedule) {
```

Comment:
This method should handle and return errors so that invokers can decide how to proceed.